internal/info/proc/devices/devices.go

```diff
@@ -67,8 +67,11 @@ func (d devices) Count() int {
 
 // Exists checks if a Device with a given name exists or not
 func (d devices) Exists(name Name) bool {
-	_, exists := d[name]
```

Comment:
I'm sure I had this in a local change. Seems I missed it with the back and forth.
We could just call:
```
_, exists := d.Get(name)
```
here.

Reply:
Sure